lib/src/util/stack_trace_mapper.dart

```diff
@@ -3,84 +3,81 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'package:collection/collection.dart';
+
 import 'package:package_resolver/package_resolver.dart';
 import 'package:source_map_stack_trace/source_map_stack_trace.dart' as mapper;
 import 'package:source_maps/source_maps.dart';
 
 /// A class for mapping JS stack traces to Dart stack traces using source maps.
 class StackTraceMapper {
   /// The parsed source map.
-  final Mapping _mapping;
+  Mapping _mapping;
```

Comment:
Mention in the documentation that this is initialized lazily in `mapStackTrace()`.